bash/save-flash-linux.sh

```diff
@@ -23,9 +24,12 @@ usage() {
     echo 'Copy a flash movie (youtube.com, etc) saved by a browser in /private'
     echo
     echo Options:
-    echo "  -n, --number NUMBER default = $number"
+    echo "  -n, --number NUMBER	default = $number"
     echo
-    echo "  -h, --help         Print this help"
+    echo "  -a, --audio		Save as MP3 File (requires ffmpeg)"
```

Comment:
Notice how the "Save as ..." description doesn't line up with the "Print this help" below. I suspect your change contains tab characters. Please use spaces instead, that way the vertical "columns" will line up wherever you view them (in the terminal, less command, diffs, etc). An easy way to replace all tabs with the right amount of spaces in the entire file is in `vim`, using the `:retab` command. Thanks.